bdb/ll.c

```diff
@@ -258,12 +258,17 @@ int ll_dta_del(bdb_state_type *bdb_state, tran_type *tran, int rrn,
     unsigned long long search_genid;
     int crc;
     int is_blob = 0;
+    int is_rowlocks = 0;
 
     if (dta_out) {
         bzero(dta_out, sizeof(DBT));
         dta_out->flags = DB_DBT_MALLOC;
     }
 
+    if (tran->logical_tran) {
```

Comment:
Do you think we need to check `dtafile == 0` here?

Reply:
Sure, that could be an optimization to not verify updateid for blobs since we already have a rowlock on this row.